### PR TITLE
Plague Scheme

### PR DESCRIPTION
**Pull request by SolTsuki**
# Changelog:
- The Lich King can start the Plague Scheme:
  - The Plague is limited to 30 counties.
  - Only humans are vulnerable to the Plague.
  - The Plague spreads to counties with high Development and prefers historical Plague regions (so it avoids Hearthglen).
  - Agents accept the Lich Ling's faith and culture after the Plague Scheme.
  - Characters become undead staying in plagued courts.
  - Plagued lands are taken with the Great Invasion CB.
  - Great Invasion war is declared when the Plague starts.
  - Characters that have Mord'truari virtues and the Cynical trait joins the Plague scheme.
  - Plagued counties give special troops based on Development.
- The Lich King trait gives siege progress to Skirmishers and Giants so they can siege too.
- Ner'zhul may declare war on Lordaeron and Quel'thalas without bordering it.
- Ner'zhul has 0 Dark Frenzy in the year 605 bookmark.
- Kel'Thuzard is cynical, ambitious, and deceitful.
- Alexei Barov is greedy, callous, and arrogant.
- Illucia is callous, arrogant, and chaste.
- Pharamond Rivendare is cynical, calm, and callous.

# Chains:
- [x] Ongoing Plague events (happens during the scheming):
  - [x] Event: You spread Death cults to help your scheme. You may choose your strategy.
    - Option 1: Trick cultists into serving, recruit cultists by intrigue
    - Option 2: Sway cultists, recruit cultists by diplomacy.
    - [x] Need text for this, 600-400 characters, from Ner'zhul perspective, in 2nd person
- [x] Plague Outbreak chain (what happens when the scheme is finished):
  - [x] Event: You orders (or may refuse) cultists to infect grain and wood sources
     - [x] Need text for this, 600-400 characters, from Ner'zhul perspective, in 2nd person
  - [x] Event: You succeed, the Plague spreads
     - [x] Need text for this, 600-400 characters, from Ner'zhul perspective, in 2nd person
     - [x] The last 150 characters should tell if the cultists were captured and told who they serve
  - [x] Event: You failed, the Plague prevented
     - [x] Need text for this, 600-400 characters, from Ner'zhul perspective, in 2nd person
     - [x] The last 150 characters should tell if the cultists were captured and told who they serve
  - [x] Event: Your lands are plagued, a notification
     - [x] Need text for this, 600-400 characters, from the victim ruler perspective, in 2nd person
     - [x] The last 150 characters should tell if the cultists were captured and told who they serve
  - [x] Event: Your guards prevented the Plague, a notification
     - [x] Need text for this, 600-400 characters, from the victim ruler perspective, in 2nd person
     - [x] The last 150 characters should tell if the cultists were captured and told who they serve
- [x] Character got the Plague started by the Plaguer and turned undead
  - [x] Need text for this, 600-400 characters, from the sick person perspective, in 1st person

# Tasks:
<details>
<summary>Click to expand</summary>

- [x] Lock the scheme to Lich King only
- [x] Cleared `block_plague_scheme`
- [x] Add the scheme game concept
- [x] Update the scheme game concept at the end
- [x] Confirm `plagued_rival_effect` works
- [x] Removed unwanted events from `plague_ongoing`
- [x] Adapt `plague_outcome.5002`
- [x] Fire `mx_cue_murder` only once, not in both event and on-action
- [x] Adapt `attempted_murder_opinion_effect`, add Plagued Me opinion
- [x] `plague_outcome.5001` tells when the Plague appears in your province instead of telling when the scheme fired
- [x] Check `wc_plague_effects`, they look half-finished
- [x] Check `wc_plague_scheme_discovery_events`, it may lack adaptation or be not-needed
- [x] Secrets don't work (or remove?)
  - [x] Adapt `secret_exposed_notification_effect`, it should send event
  - [x] Adapt `secret_exposed_owner_effects_effect`, it should add opinion
- [x] Move scheme changes from the Stun branch
- [x] `spread_plague_barony` may stop with the character death
- [x] Fix these:
<details>
<summary>Click to expand</summary>

image

</details>

- [x] Adapt this:
  - [x] Ner'zhul shouldn't be able to draw a map himself
  - [x] You don't need to draw a map of Terenas' castle since the scheme targets different barony
<details>
<summary>Click to expand</summary>

image

</details>

- [x] Revert test changes in the plague and murder schemes
- [x] Those who joined the Plague scheme convert to Lich King's religion once the scheme is over
- [x] Stop Plague at some point
- [x] Add Plague feature to Lich King trait
- [x] Make `on_invalidated`
- [x] Scourge declare war
- [x] Check `scheme` error
- [x] Owner to reveal not showing
<details>
<summary>Click to expand</summary>

image

</details>

- [x] Ner'zhul followers shouldn't have negative reaction to Plague
- [x] Add scheme cooldown in 603
- [x] Plague scheme has more agents but they give less progress
- [x] The Scourge not getting a notification on historical Plague
- [x] Make Plague work without Plaguer
- [x] Create Plague CB
</details>

# Tests:
- [x] Make sure everything works when the scheme is discovered

## How to test:
Start Ner'zhul in the year 603, right-click yourself, Debug Interactions, Give Dark Frenzy a few times so you have enough to start the Plague.

Right-click your target, Start a Plague.

`event WCHEA.1001` to get the Plague of Undeath.

To see the Plague spreading open either the Culture or Faith maps, Plagued provinces will have the Scourge culture and Mord'truari faith.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/647/files?authenticity_token=yd8G%2FHS%2BmXG01lp9JVYPet8MR4u4q9vH%2FuySfJKS%2BpPy9cyohgddB%2Bac4erFGpM7ffICnwv2br%2BBEGzOAXkKMg%3D%3D&file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

### CODE REVIEW

File: common/deathreasons/wc_plague_deaths.txt
Comment (zumbak04) [inline common/deathreasons/wc_plague_deaths.txt:6]: First, noticed anomaly line changes.
<details>
<summary>Click to expand</summary>

image

</details>

You copy vanilla files and comment on them. Clear it.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/deathreasons/wc_plague_deaths.txt
Comment (SolTsuki) [reply common/deathreasons/wc_plague_deaths.txt]: Cleaned up.

File: events/scheme_events/wc_plague_scheme/wc_plague_saves/wc_plague_save_events.txt
Comment (zumbak04) [inline events/scheme_events/wc_plague_scheme/wc_plague_saves/wc_plague_save_events.txt:198]: Again clearing the branch. Commented out vanilla events.

File: events/secret_events/wc_plague_secrets_events.txt
Comment (zumbak04) [inline events/secret_events/wc_plague_secrets_events.txt:8]: Why do you need these events? Need your answer before I can suggest anything.

Comment (SolTsuki) [reply events/secret_events/wc_plague_secrets_events.txt:8]: I copied those from the murder scheme, but because they needed a lot of extra code to make them work (Plaguer trait for instance) and wasn't sure it made sense, I just left those there for now. Basically, if you try to plague/manage to plague someone's lands should it be treated like a murder? like: there's a secret, if the secret is found out you can be blackmailed... I don't think the Plaguer trait makes that much sense, but I'm not sure about the whole secret thing, what do you think?

File: events/scheme_events/wc_plague_scheme/wc_plague_saves/wc_plague_save_events.txt
Comment (SolTsuki) [reply events/scheme_events/wc_plague_scheme/wc_plague_saves/wc_plague_save_events.txt]: Done, also deleted now, didn't realize those were there

File: events/secret_events/wc_plague_secrets_events.txt
Comment (zumbak04) [reply events/secret_events/wc_plague_secrets_events.txt:8]: If they're vanilla copied and just need a few changes.
You may delete these and tweak vanilla events to work for both murder and plague:
- Set a scope value to define if it's murder or plague (the scope value will be accessible in the event chain)
- Add text variations (descs, options, etc.) for plague

> I don't think the Plaguer trait makes that much sense, but I'm not sure about the whole secret thing, what do you think?

It depends. If it's used only by Ner'zhul, prob not.

Comment (SolTsuki) [reply events/secret_events/wc_plague_secrets_events.txt:8]: > If they're vanilla copied and just need a few changes.

I'll try that, they're almost the same so it shouldn't be hard.


> It depends. If it's used only by Ner'zhul, prob not.

Right now it's just for him so for now I'll keep it simple and won't add any special trait, if later we find out it's a good idea for other characters to do plagues we could add it.

File: localization/english/wc_purge_l_english.yml
Comment (zumbak04) [inline localization/english/wc_purge_l_english.yml:3]: OUR OBJECTIVE? Don't get.

Comment (SolTsuki) [reply localization/english/wc_purge_l_english.yml:3]: it's a placeholder, gonna use the purged barony's name.

Comment (zumbak04) [reply localization/english/wc_purge_l_english.yml:3]: > it's a placeholder, gonna use the purged barony's name.

Better localize it now not to forget later.

File: common/on_action/wc_plague_on_actions.txt
Comment (zumbak04) [inline common/on_action/wc_plague_on_actions.txt:1]: Could you show the plan for purge events?

Comment (zumbak04) [reply common/on_action/wc_plague_on_actions.txt:1]: There are 2 files named the same. Need to change it.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/on_action/wc_plague_on_actions.txt
Comment (SolTsuki) [reply common/on_action/wc_plague_on_actions.txt:1]: Purge events are divided in 3 categories: starting, purge, ending. Starting events are the ones that are related to getting inside the plague barony (something like finding an Ogre camp or a savage beast), purge events are the ones related to the purgiong activity (killing undead, burning stuff...) and the ending is a finale, like, fighting some boss (abomination, lich). You have to suceed in some of the events in each stage to get to the next one.
Also, if a war happens, the "starting events" won't be needed because you are sort of already there.

Comment (SolTsuki) [reply common/on_action/wc_plague_on_actions.txt:1]: Cleaned up, there's no files with same name anymore.

Comment (zumbak04) [reply common/on_action/wc_plague_on_actions.txt:1]: > Cleaned up, there's no files with same name anymore.

Not these files, they can't be merged, they're from different folders.
<details>
<summary>Click to expand</summary>

image

</details>

I meant the on action files, `wc_plague_on_actions.txt`.

Comment (zumbak04) [reply common/on_action/wc_plague_on_actions.txt:1]: > fighting some boss (abomination, lich)

Hmm? Who fights? If it's just one character, what is army doing at that moment?

File: events/decisions_events/wc_purge_events.txt
Comment (zumbak04) [inline events/decisions_events/wc_purge_events.txt:6]: The purge should be on a separate branch from the basic Plague mechanics.
When you mix different things on one branch, you become difficult to both design and test.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: events/decisions_events/wc_purge_events.txt
Comment (SolTsuki) [reply events/decisions_events/wc_purge_events.txt:6]: How can you test purging mechanics if you have no plagued barony?

File: events/scheme_events/wc_plague_scheme/wc_plague_ongoing_events.txt
Comment (zumbak04) [inline events/scheme_events/wc_plague_scheme/wc_plague_ongoing_events.txt:19]: The text is almost the same as vanilla event.
<details>
<summary>Click to expand</summary>

image

</details>

That's what I was [talking about](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/647#discussion_r702279914). You don't need to copy a vanilla event if you just change one localization line.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Comment by AScic1** [comment]
Was able to use console commands to set multiple plagues, one on Kul Tiras, one in Bleeding Hallow and one failed in Grizzlemaw Tribe. Everything appeared to work correctly as it gave me failure or success text. 

Checked Kul TIras and Bleeding Hallow counties and saw the mechanics in progress, changing faith, culture, and giving the different modifiers.

**Review by ValianBlue — CHANGES_REQUESTED** [review]
**Requested Changes**

_In multiple instances, I was unable to use the "Start a Plague" interaction due to "None of [X]'s counties are vulnerable to the Plague." While this is noted in the tooltip for the Plague itself, I would like to see an additional note added to the Start a Plague tooltip that informs the player what is preventing the interaction._

<details>
<summary></summary>

image
</details>

_I find it odd that characters who completely despise the Lich King like Jaina join his scheme as agents to spread the plague. A significant negative penalty should be applied for the scheme itself if possible. If not, the Scourge culture or Mord'truari religion could work as well._

<details>
<summary></summary>

image
</details>

_On my first attempt to plague Lordaeron, I had a 95% chance to succeed and a 95% of plot secrecy. This made the scheme trivial to execute, though in my attempts to reproduce this I found it was 44% and 59% on the second attempt. 95% and 95% on the third attempt. 79% and 94% on the fourth. The variability appears to be based on the potential agents that can join the plot. Adjusting the chance of agents joining the plot should resolve this to be more consistent and realistic._

_Perhaps an additional mechanic could be added to secretly convert rulers in foreign realms. You can sort of accomplish this in the base game, but it's not exactly straightforward._

<details>
<summary></summary>

image
</details>

<details>
<summary></summary>

image
</details>

<hr>

**Testing Observations**

_Executing the plague was very straightforward and worked successfully in converting the religion and culture of the target province. I found that targeting Stratholme gave me an army of 1700 to command._

<details>
<summary></summary>

image
</details>

_This event was a very nice touch, nicely done._

<details>
<summary></summary>

image
</details>

**Comment by zumbak04** [comment]
> In multiple instances, I was unable to use the "Start a Plague" interaction due to "None of [X]'s counties are vulnerable to the Plague." While this is noted in the tooltip for the Plague itself, I would like to see an additional note added to the Start a Plague tooltip that informs the player what is preventing the interaction.

Added in f91ec015750b054c7f18e2dba0cf169947c6c130.

> I find it odd that characters who completely despise the Lich King like Jaina join his scheme as agents to spread the plague. A significant negative penalty should be applied for the scheme itself if possible. If not, the Scourge culture or Mord'truari religion could work as well.

Changed in 237076b77592d98c56734bea0130f73c1d651351.

> Perhaps an additional mechanic could be added to secretly convert rulers in foreign realms. You can sort of accomplish this in the base game, but it's not exactly straightforward.

Don't get what you mean.

**Comment by ValianBlue** [comment]
>_Perhaps an additional mechanic could be added to secretly convert rulers in foreign realms. You can sort of accomplish this in the base game, but it's not exactly straightforward._

> Don't get what you mean.

_Conversion as in a religious conversion primarily to the Mord'truari faith. The scheme would be to "Convert [X-Character] to Our Faith], and on a successful attempt the target secretly converts to Mord'truari. It'd be representative of how the Cult of the Damned was able to proliferate. On a different note, perhaps the societies mechanic from CK2 could be used instead for this purpose, though I have a feeling we might see it again in a CK3 DLC._

**Comment by zumbak04** [comment]
> It'd be representative of how the Cult of the Damned was able to proliferate.

This function is represented by...

> Agents accept the Lich Ling's faith after the Plague Scheme.

**Comment by ValianBlue** [comment]
_Excellent. I was originally referring to being able to convert them **before** the scheme, so that they would be far more willing to join as an agent. Looking over this in-game though, it works well enough to get a strong hook on someone and to force them to join your scheme. Perhaps it's bending the power of the strong hook (or even just a bribe) a bit to forcefully convert someone following the Scheme plague, though._

**Review by CrookDoodle — APPROVED** [review]
No problem in the proofreading department.

**Review by TheGroxEmpire — COMMENTED** [review]
- There could be a better way to educate player about this plague scheme.
Suggestion: In lore, Ner'zhul tested the plague on human inhabitants of Northrend before releasing it to Lordaeron (https://wowpedia.fandom.com/wiki/Icecrown_and_the_Frozen_Throne). We can take this lore precedent to make an additional event to introduce this plague mechanics to players. Starting the game on 603, the player would have an event storyline where Ner'zhul begins developing the Plague of Undeath. After the event culmination where the human settlement of Northrend has been devastated, an event may pop-up and player would be notified that he'd be able to scheme plague to another Human settlement

- This last option on the event ``plague_maintenance.100`` should have a tooltip saying that a player would not be able to scheme plague until 5 years has passed. It wasn't clear for me at first that it'd cancel the plot instead of allowing me to choose different target.
<details>

image
</details>

- Arthas is very approving of Scourge virtues. The only thing that stops him from joining the plague scheme is that he likes his dad. This is very volatile and could maybe change in different playthrough. It is very lore breaking and may ruin future event for him on some playthrough:
<details>

image
</details>

- Would it be possible to change the win demands to not convert the ruler of the land to undead and have them run instead? It would convert some lore important characters, making them out of use for future events.
<details>

image
</details>

**Comment by zumbak04** [comment]
> We can take this lore precedent to make an additional event to introduce this plague mechanics to players. Starting the game on 603, the player would have an event storyline where Ner'zhul begins developing the Plague of Undeath.

The Plague of Northrend happened before the year 603.

**Comment by zumbak04** [comment]
> It is very lore breaking and may ruin future event for him on some playthrough:

-35 is a lot. I design events so it won't hurt anything.

> This last option on the event plague_maintenance.100 should have a tooltip saying that a player would not be able to scheme plague until 5 years have passed. It wasn't clear for me at first that it'd cancel the plot instead of allowing me to choose different target.

Changed in 9c9c25dd0b32f87cf4cce19e44202f39763c1aa0.

**Comment by zumbak04** [comment]
> Would it be possible to change the win demands to not convert the ruler of the land to undead and have them run instead? It would convert some lore important characters, making them out of use for future events.

If you meant Uther and Garithos, I changed it in 0aa0587d7b5b2184b472bbaa322ae437219035f7 and 92d29c2d20666b19d7d0d2cef7f915a25ab91fad.

**Review by Melnerag — COMMENTED** [review]
### Plague Wars not always Starting

I've initiated a plague in the South Tract, and Lordaeron & Dalaran got plagued and correctly pulled into the war. As you can see in the image below, the Plague also spread to Gilneas and Stromgarde, but these two realms did not have any wars declared on them, even though event troops were correctly spawned in the provinces. 

<details>
<summary>Plague Spread & Wars</summary>

image
</details>

### Scheme Discovered
Ayep, it still works if discovered. King Terenas has discovered the Plague, correctly reducing the success chance. The Plague still went through on 2nd attempt after the 1st one failed.  

### Overall First Impressions
The scheme works well, with the right individuals joining in. The plague spreads wonderfully, infecting human areas and avoiding other races. I feel that perhaps once the Plague triggers it should be obvious who did it, instead of it remaining a secret. And there should be a heavier 'cost' to failing the scheme, as right now it was all too easy to try again with a pretty high success chance. ('If at first you don't succeed' option)

**Comment by zumbak04** [comment]
> I've initiated a plague in the South Tract, and Lordaeron & Dalaran got plagued and correctly pulled into the war. As you can see in the image below, the Plague also spread to Gilneas and Stromgarde, but these two realms did not have any wars declared on them, even though event troops were correctly spawned in the provinces.

Fixed in 075772911dd47c143969cd4e04553cc5fc1d96f3.

> I feel that perhaps once the Plague triggers it should be obvious who did it, instead of it remaining a secret.

Like in the lore where no one knew Ner'zhul was behind the Plague.

> And there should be a heavier 'cost' to failing the scheme, as right now it was all too easy to try again with a pretty high success chance. ('If at first you don't succeed' option)

Don't exactly get what you want, but I increased the after-scheme resistance in 87af0a3d2367e9dcfa8007a99f7c48fa7a438190.

**Review by TheGroxEmpire — COMMENTED** [review]
- If target is at truce, successfully scheming plague would not launch invasion:
<details>

image
</details>

Launching plague near another ruler counties made their counties Scourge cultured. This caused a weird problem where you could not invade Gilneas after invading Lordaeron, without Gilneas being plagued at all:

<details>
 
image


image

</details>

File: localization/english/wc_game_concepts_l_english.yml
Comment (arithon) [inline localization/english/wc_game_concepts_l_english.yml:99]: [plagued|E] leads to the Plague Scheme, so a link in this text is unnecessary.

**Review by arithon — COMMENTED** [review]
apart from the comment, I think the loc is fine (at least what was changed in this PR)...

**Review by Melnerag — COMMENTED** [review]
### Plague Wars
Retested Plague wars by starting it in the South Tract. Lordaeron, Dalaran, Gilneas, the Syndicate and Ravenholdt were all correctly  had war declared on them once their lands became plagued. The Plague also spread to one border county in Stromgarde, but Stromgarde did not have a war declared on it despite a plague army spawning in the county. 

Is this intended behavior to exclude Stromgarde?

### Plague Spread
I tried plaguing Kul Tiras. The three main islands got correctly plagued, and the plague did not spread to any of the channel islands or Tol Borad, and also correctly avoided the Quilboar and Gnome cultured counties.  Working as intended. The options for plague origin counties also made sense.

### Modifier after Plague
A new more severe modifier is applied after the Plague finishes. All's well.

### Future-proof?
I tried reviewing the code, but I am not familiar enough with CK stuff to be too certain. I don't know what the intention is, and if the Scourge is supposed to eventually be able to learn to Plague other races, but if that is the intention then perhaps now is a good time to future-proof it instead of hardcoding the Plague to only affect humans? If it is only ever intended to work on humans, then all's well.

---
Overall, amazing job. I don't understand how the code works, but it does, and does so without slowing the game down!

**Comment by zumbak04** [comment]
> * If target is at truce, successfully scheming plague would not launch invasion:

I know. Do you suggest a change?

> Launching plague near another ruler counties made their counties Scourge cultured. This caused a weird problem where you could not invade Gilneas after invading Lordaeron, without Gilneas being plagued at all:

I don't get what you mean. Found this CB available.
<details>
<summary>Click to expand</summary>

image

</details>

File: localization/english/wc_game_concepts_l_english.yml
Comment (zumbak04) [reply localization/english/wc_game_concepts_l_english.yml]: Fixed in 86637f176b523b950f7f3cd09b2c65c15d8391b7.

**Comment by zumbak04** [comment]
> Stromgarde did not have a war declared on it despite a plague army spawning in the county.

It could be called in a war against the Scourge via alliances. Thus a new war isn't possible.

> hardcoding the Plague to only affect humans?

It's not hardcoded. We may easily change it.

**Comment by TheGroxEmpire** [comment]
> If target is at truce, successfully scheming plague would not launch invasion:

I think truce should be disabled for Scourge if possible.

> I don't get what you mean. Found this CB available.

When I chose the third most populous city to plague in Lordaeron. It caused Gilneas county to be Scourge cultured. It disabled me to plague Genn as there's no human cultured county under him. Normal CB still works, just the scheme that doesn't.

**Comment by zumbak04** [comment]
> I think truce should be disabled for Scourge if possible.

Having a truce is essential. It gives time for war losers.

> When I chose the third most populous city to plague in Lordaeron. It caused Gilneas counties to be Scourge cultured. It disabled me from doing plague on Genn as there's no human cultured county under him. Normal CB still works, just the scheme that doesn't.

It should be disabled because there's nothing to plague.

**Review by TheGroxEmpire — APPROVED** [review]
All is considered then.

**Review by arithon — APPROVED** [review]
loc lgtm then

**Review by Melnerag — COMMENTED** [review]
### Cultist Event outcome effect names misleading
See below. While it is the cultists who do the infiltrating and the overwhelming of the locals, the modifiers are instead called 'Cultists Infiltrated' and 'Cultists Overwhelmed'. 

Perhaps rename them to 'Cultist Infiltration/Infiltrators' and 'Locals Overwhelmed'?

<details> 
<summary> Who overwhelms/infiltrates whome? </summary>

image
 </details>

### Confirmed Plague Wars declaring
I ran a few more tests, and Stromgarde is indeed getting correctly declared against. All is well.

### Plague Dificulty
I gave myself the Island of Balor  to reach all the humans and see what is the base Plague chance. This is the scheme power without me bribing anybody.

Stormwind: 95%,
Bloodsail: 95%
Kul Tiras: 95%
Gilneas: 95%
Stromgarde: 78%
Dalaran: 67%
Syndicate: 88%

I noticed that having just about trait like cynical, arbitrary or sadistic was enough for them to join even if they had virtues like honest or compassionate. I think ValianBlue is onto something when he said that the Plague Scheme could be too easy.

**Comment by zumbak04** [comment]
> See below. While it is the cultists who do the infiltrating and the overwhelming of the locals, the modifiers are instead called 'Cultists Infiltrated' and 'Cultists Overwhelmed'.

Yes. Cultists Infiltrated/Overwhelmed Lordaeron/locals/county. I don't get your point.

> Perhaps rename them to 'Cultist Infiltration/Infiltrators' and 'Locals Overwhelmed'?

Locals overwhelmed? Overwhelms what?

> I noticed that having just about trait like cynical, arbitrary or sadistic was enough for them to join even if they had virtues like honest or compassionate. I think ValianBlue is onto something when he said that the Plague Scheme could be too easy.

Made it harder in 1fad23302962c93909bc765041f0af49594978ce.

**Review by Melnerag — COMMENTED** [review]
I will defer to the English Proofreader regarding 'Infiltrated' and 'Overwhelmed'.

### Plague Scheme Balance
Scheme power is much improved, it is still possible to find agents with bribes, but at least the base Plague chance is much lower. 

One oddity I still saw is that Zealous characters would still accept the scheme. Is it possible to make Zealous non-Mord'truari much less likely to join?

<details>
<summary> Two zealots who joined in </summary>

image

image

</details>

**Comment by zumbak04** [comment]
> One oddity I still saw is that Zealous characters would still accept the scheme. Is it possible to make Zealous non-Mord'truari much less likely to join?

Changed in c54f2cefd2ae6fad12a1b487fc14683e1081f01e.

**Review by Melnerag — APPROVED** [review]
Retested & Approved! Did not see any other significant outliers while testing.

One minor remark, Pharamond Rivendare will not accept the Plague scheme even with a bribe. It doesn't matter in the grand scheme of things I think, as in Historical Plagues branch he will be treated as having joined it. But if it is important that he join, here are his reasons:

image

**Comment by zumbak04** [comment]
> One minor remark, Pharamond Rivendare will not accept the Plague scheme even with a bribe. It doesn't matter in the grand scheme of things I think, as in Historical Plagues branch he will be treated as having joined it. But if it is important that he join, here are his reasons:

I'll open a suggestion, Terenas is a too good ruler. Almost all the vassals have 100 opinion.

**Pull request merged**
